connect_ext_ppr/webapp.py

```diff
@@ -37,6 +39,37 @@
 )
 class ConnectExtensionXvsWebApplication(WebApplicationBase):
 
+    @router.get(
```

Comment:
could you please add some test cases?

Reply:
Yes, but we have separate task for it to not block the development

Reply:
@d3rky 
do you agree with this?
If yes, please specify the ticket for that and resolve the comment.

Reply:
Decided to add tests there

Reply:
Added